admin-dev/themes/new-theme/js/app/pages/stock/store/actions.js

```diff
@@ -24,14 +24,14 @@
  */
 import Vue from 'vue';
 import VueResource from 'vue-resource';
-import * as types from './mutation-types';
-import { showGrowl } from 'app/utils/growl';
-import { EventBus } from 'app/utils/event-bus';
+import {showGrowl} from 'app/utils/growl';
+import {EventBus} from 'app/utils/event-bus';
```

Comment:
Can we activate space-in-bracket ? That looks ugly and less readable to my opinion

Reply:
haha, totally the opposite for me :sweat_smile:

Reply:
Erf... Let's use Pablo to decide ahah

Reply:
He said, use airbnb rules :)